src/app/android/app/src/debug/AndroidManifest.xml

```diff
@@ -0,0 +1,7 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android">
+    <!-- The INTERNET permission is required for development. Specifically,
+         the Flutter tool needs it to communicate with the running application
+         to allow setting breakpoints, to provide hot reload, etc.
+    -->
+    <uses-permission android:name="android.permission.INTERNET"/>
```

Comment:
Why do we need to copy this permission here? Why is the main manifest not enough?

Reply:
It seems like the main manifest doesnt do anything

Reply:
The app wouldnt deploy with the debugger, and when you opened it without, it would not have access to the internet

Reply:
Fixed main is used a based and the other one are merged into it.